### PR TITLE
Feature/auto switch

### PR DESCRIPTION
**Pull request by KaidenR**

### CODE REVIEW

**Comment by KaidenR** [comment]
@parkerholladay when you get a sec, can you take a look at the way I updated the install to update old post-commit hooks in the repos?

File: src/client/services/user.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export function rotate() {
     ...inactiveUsers
   ]
 
+  console.log(`rotating to ${updatedUsers.filter(u => u.active).map(u => u.name).join(', ')}`)
```
Comment (parkerholladay) [inline src/client/services/user.js]: Remove this log?

File: src/package.json [hunk 2]
```diff
@@ -25,8 +25,9 @@
     "babel-preset-stage-0": "6.22.0",
     "babel-register": "6.24.1",
     "babel-require": "1.0.1",
-    "electron": "1.4.12",
-    "menubar": "5.2.1"
+    "electron": "^1.7.9",
+    "menubar": "5.2.1",
+    "uuid": "^3.1.0"
```
Comment (parkerholladay) [inline src/package.json]: Should we make these exact?

File: src/utils/install.js [hunk 3]
```diff
@@ -24,6 +28,19 @@ export default function() {
...
     fs.writeFileSync(CONFIG_FILE, JSON.stringify({ users: [], repos: [] }), 'utf-8')
 
-  if (!fs.existsSync(POST_COMMIT_FILE))
-    fs.writeFileSync(POST_COMMIT_FILE, POST_COMMIT_GIT_SWITCH, 'utf-8')
+  installPostCommitHook()
+}
+
+function installPostCommitHook() {
+  const alreadyInstalled = fs.existsSync(POST_COMMIT_FILE) &&
+    fs.readFileSync(POST_COMMIT_FILE, 'utf-8') === POST_COMMIT_GIT_SWITCH
```
Comment (parkerholladay) [inline src/utils/install.js]: This looks good. I like it.

**Pull request merged**